test/spec/features/popup-menu/ReplaceMenuProviderSpec.js

```diff
@@ -511,6 +511,24 @@ describe('features/popup-menu - replace menu provider', function() {
       }));
 
 
+      it('should set loop characteristics type', inject(function(bpmnReplace, elementRegistry) {
+
+        // given
+        var task = elementRegistry.get('LoopTask'),
+            businessObject = getBusinessObject(task);
+
+        openPopup(task);
+
+        // when
+        triggerAction('toggle-parallel-mi');
+
+        // then
+        var newLoopCharacteristics = businessObject.loopCharacteristics;
+
+        expect(is(newLoopCharacteristics, 'bpmn:MultiInstanceLoopCharacteristics')).to.be.true;
```

Comment:
Should we also verify the value of `isSequential`?

Reply:
Although that wasn't broken, it doesn't hurt to add an extra layer to the tests